tests/gpu/GPUUnitTest.cpp

```diff
@@ -228,8 +228,13 @@ namespace
             // It means to generate the input values.
 
             const bool testWideRange = test->getTestWideRange();
+#if __aarch64__
```

Comment:
The define is for ARM processors only. Should that be `__aarch64__ && __APPLE__` ?

Reply:
yeah. Better to limit it to Apple.